### PR TITLE
[ESIMD] Update usage of deprecated block_load/store APIs

### PR DESCRIPTION
**Pull request by DenisBakhvalov**
Part1. Automated updates.

### CODE REVIEW

File: SYCL/ESIMD/accessor.cpp [hunk 1]
```diff
@@ -53,7 +53,8 @@ int main() {
             using namespace sycl::ext::intel::experimental::esimd;
             unsigned int offset = 0;
             for (int k = 0; k < VL / 16; k++) {
-              simd<Ty, 16> var = block_load<Ty, 16>(acc0, offset);
+              simd<Ty, 16> var;
+              var.copy_from(acc0, offset);
```
Comment (kbobrovs) [inline SYCL/ESIMD/accessor.cpp:57]: please change block_store to copy_to below too

Comment (DenisBakhvalov) [reply SYCL/ESIMD/accessor.cpp:57]: This work is in progress.
I only uploaded easy replacements that I've done with a script.
Changing all the places might take a while, so I started pushing patches.

Comment (DenisBakhvalov) [reply SYCL/ESIMD/accessor.cpp:57]: Ok, never mind. I replaced all the other occurrences as well.

**Review by kbobrovs — DISMISSED** [review]

**Comment by vladimirlaz** [comment]
@kbobrovs, could you please review/approve the PR?

**Review by kbobrovs — APPROVED** [review]

**Pull request merged**